src/main/java/com/hm/achievement/lang/command/InfoLang.java

```diff
@@ -0,0 +1,35 @@
+package com.hm.achievement.lang.command;
+
+import com.hm.achievement.lang.Lang;
+
+public enum InfoLang implements Lang {
+	DESCRIPTION("Description:"), 
+	DESCRIPTION_DETAILS("Advanced Achievements enables unique and challenging achievements. " +
+			"Try to collect as many as you can, earn rewards, climb the rankings and receive RP books!"), 
+	VERSION("Version:"), 
+	AUTHOR("Author:"), 
+	WEBSITE("Website:"), 
+	VAULT("Vault integration:"), 
+	PETMASTER("Pet Master integration:"), 
+	BTLP("BungeeTabListPlus integration:"), 
+	ESSENTIALS("Essentials integration:"), 
+	PLACEHOLDERAPI("PlaceholderAPI integration:"), 
+	DATABASE("Database type:");
+
+	private final String defaultMessage;
+
+	InfoLang(String defaultMessage) {
+		this.defaultMessage = defaultMessage;
+	}
+
+
```

Comment:
Double blank lines. There are also similar occurrences in some of the other _Lang_ implementations. ^^